### PR TITLE
Progress linear docs update

### PR DESCRIPTION
**Pull request by DmitriiBabakaevTR**

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -2631,41 +2631,60 @@ __Progress circular__ - The v-progress-circular component is used to convey data
...
-      bufferValueProgressBars.get(index).is().reactive();
-      bufferValueProgressBars.get(index).is().determinate();
-      bufferValueProgressBars.get(index).has().color(color);
-      bufferValueProgressBars.get(index).click();
-      bufferValueProgressBars.get(index).has().value(50.0);
-    }
-
+@Test(description = "Test checks absolute value of progress linear and checks if it is hidden : value")
+public void absoluteValueHiddenProgressLinearTests() {
+  refresh();
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2642]: please, correct

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -2631,41 +2631,60 @@ __Progress circular__ - The v-progress-circular component is used to convey data
...
+  <div class="v-progress-linear__stream red--text text--lighten-2" style="width: 100%;"></div>
+  <div class="v-progress-linear__background red lighten-2" style="opacity: 0.3; left: 0%; width: 0%;"></div>
+  <div class="v-progress-linear__buffer" style="width: 0%;"></div>
+  <div class="v-progress-linear__determinate red lighten-2" style="width: 0%;"></div>
+</div>
+```
+
+|Method | Description                          | Return Type
+--- |--------------------------------------| ---
+**is()** | Returns Assert class                 | ProgressBarAssert
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2675]: check, please, it should be ProgressLinearAssert

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 1 above)
Comment (DmitriiBabakaevTR) [reply source/includes/documentation/_vue_js_elements.md]: done

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (DmitriiBabakaevTR) [reply source/includes/documentation/_vue_js_elements.md]: done

**Review by MayaElf — APPROVED** [review]

File: source/includes/documentation/_vue_js_elements.md [hunk 3]
```diff
@@ -2631,41 +2631,59 @@ __Progress circular__ - The v-progress-circular component is used to convey data
...
-    public void bufferValueProgressBarsTests(int index, String color) {
-      bufferValueProgressBars.get(index).is().displayed();
-      bufferValueProgressBars.get(index).is().reactive();
-      bufferValueProgressBars.get(index).is().determinate();
-      bufferValueProgressBars.get(index).has().color(color);
-      bufferValueProgressBars.get(index).click();
-      bufferValueProgressBars.get(index).has().value(50.0);
-    }
-
+@Test(description = "Test checks absolute value of progress linear and checks if it is hidden : value")
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2640]: Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby
image

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 3 above)
Comment (DmitriiBabakaevTR) [reply source/includes/documentation/_vue_js_elements.md:2644]: added

**Review by MayaElf — APPROVED** [review]

**Pull request merged**